app/uk/gov/hmrc/traderservices/controllers/TraderServicesFrontendController.scala

```diff
@@ -254,11 +267,23 @@ class TraderServicesFrontendController @Inject() (
     }
 
   // GET /pre-clearance/import-questions/vessel-info
-  val showAnswerImportQuestionsVesselInfo: Action[AnyContent] =
-    actionNotYetImplemented
+  val showAnswerImportQuestionsMandatoryVesselInfo: Action[AnyContent] =
```

Comment:
As far as I'm aware there will be no mandatory vessel page for imports at all as the only option that now impacts it is C1601 which is only ever an option on the export journey. 

The three versions of vessel page are 

Export - Mandatory (C1601)
Export - Optional (any other options)
Import - Optional in all circumstances

I have no idea if this will change in the future but this is my current understanding. If you like we can get confirmation from the wider team.

Reply:
fair enough